src/index.html

```diff
@@ -17,6 +17,82 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="http://localhost:8080/">
+        <img
+          src="./images/logo.png"
+          alt="moyo"
+        />
+      </a>
+      <nav>
+        <ul class="container">
+          <li>
+            <a
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link"
```

add classes and use them for styling (Don't use tags for styling)